packages/x-components/tests/e2e/cucumber/sliding-panel.feature

```diff
@@ -11,15 +11,19 @@ Feature: Sliding panel component
     When  "<query>" is searched
     Then  "right" sliding panel arrow is displayed
     And   only some related tags are visible
-    When  right sliding panel arrow is clicked
+    When  "right" sliding panel arrow is clicked
     And   wait for the movement of the elements inside the sliding panel
```

Comment:
If we use the Javi's approach we can remove the wait step and simplify a lot the scenario.